app/executor.php

```diff
@@ -423,17 +423,19 @@ function (string $runtimeId, array $vars, string $data, $timeout, $activeRuntime
             $errNo = -1;
             $executorResponse = '';
 
+            $timeout = $timeout ?? (int) App::getEnv('_APP_FUNCTIONS_TIMEOUT', 900);
```

Comment:
You can use a short syntax here:

```suggestion
            $timeout ??= (int) App::getEnv('_APP_FUNCTIONS_TIMEOUT', 900);
```

Reply:
done 👍